cluster/gce/upgrade.sh

```diff
@@ -76,6 +76,9 @@ function upgrade-master() {
   detect-master
   parse-master-env
 
+  MASTER_OS_DISTRIBUTION=$(get-node-os "${MASTER_NAME}")
```

Comment:
The env is always defined as default os distro, we just override it here. But yeah, we can add a flag.